### ISSUE TITLE
[MSG] Decide on a back button for Inbox

### ISSUE THREAD

**Issue by sarahscott** [artsy/emission#669]
Are we going to use the existing round one or move toward this style?

cc @katarinabatina 

<img width="380" alt="screen shot 2017-07-10 at 10 49 19 am" src="https://user-images.githubusercontent.com/2712962/28031845-6ef431d8-655d-11e7-8ade-be767dab2398.png">

**Comment by katarinabatina** [comment]
@sarahscott yes we should but it includes getting the sticky header implemented as well 
![image](https://user-images.githubusercontent.com/6482926/28068023-c1c1f58a-6611-11e7-9461-53f3b79c9f22.png)

**Comment by orta** [comment]
This might need to go in Eigen and use the `ARSerifNavigationController`

**Comment by sarahscott** [comment]
@katarinabatina the header should be sticky now, and @orta it will indeed require eigen work. @alloy this hasn't been factored into our time estimates - should it be added to the list of necessary specifications for v1?

**Comment by alloy** [comment]
Yep.

Btw I think the idea isn’t as much that we add a sticky header in the RN view, but that this is the navigation bar of the nav controller @orta refers to.

**Comment by katarinabatina** [comment]
@sarahscott the header shouldn't be sticky "now", it's always been that way but let me know if a separate ticket should be written for it.

**Comment by orta** [comment]
It might be worth moving that navigation controller into https://github.com/artsy/extraction so that the behaviour can be the same between Emission/Eigen

**Comment by mennenia** [comment]
for reference: some faff I put into the eigen part of this (only temp fix) in [this pr](https://github.com/artsy/eigen/pull/2390) and also explained [here](https://github.com/artsy/eigen/issues/2389).

**Comment by katarinabatina** [comment]
@sarahscott before starting on any home work this is an obvious UI thing we should wrap up before moving from messaging.
